### DISCUSSION TITLE
Wrap search in new UI or enable old UI

### DISCUSSION THREAD

**Discussion by LucioMenci** [dbeaver/dbeaver#36309] [Q&A] [answered]
The new Search UI is cool, but it doesn't have all old features. I cannot use any more ctrl+K with wrap search. I don't know if there is that option somewhere or if it is impossible to have now.

Can you put the wrap search into the new UI? Or can you create a menu voice to have the old UI?

Thank you.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @LucioMenci,

This feature comes from the platform DBeaver is built on, Eclipse; we can't modify it in any way.

It can be toggled here:

![image](https://github.com/user-attachments/assets/38233979-bd1d-483b-94d5-6464b4b8698a)
